infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/runtime/CorfuLogReplicationRuntime.java

```diff
@@ -247,6 +252,9 @@ private void consume() {
             try {
                 LogReplicationRuntimeState newState = state.processEvent(event);
                 if (newState != null) {
+                    if (newState.getType() == LogReplicationRuntimeStateType.STOPPED) {
+                        removeClusterInfoFromStatusTable(remoteClusterId);
```

Comment:
wouldnt it be better to move this inside the stop() method?  The above check will also not be needed then.

Reply:
This logic has been changed